Regular Expressions/RegExp.sublime-syntax

```diff
@@ -168,29 +67,9 @@ contexts:
     - match: '\?(?=\(\?)'
       scope: keyword.other.conditional.regexp
       set: [group-body, base-group]
```

Comment:
I don't think this multi-set is required here, because they are literly the same, except `group-body`  consumes the closing paren when popping.

```suggestion
      set: group-body
```